robot/basestation/templates/AutomaticControls.html

```diff
@@ -0,0 +1,111 @@
+<!-- Automatic controls section -->
```

Comment:
If we're gonna argue semantics, as far as I'm concerned (I may be wrong) "manual controls" the way we have implemented isn't exactly FK since we're not actually calculating the position in a grid of the end-effector after moving the joints however much, right?